packages/components/app/styles/components/pagination.scss

```diff
@@ -4,25 +4,56 @@
 
 @use "../mixins/focus-ring" as *;
 
+// TODO: Move to pagination.json tokens
+:root {
+  --token-pagination-child-spacing-vertical: 8px;
+  --token-pagination-child-spacing-horizontal: 20px;
+}
+
 
 // * Pagination Parent (wrapper) component
 // -------------------------------------------------------------------
 
 .hds-pagination {
   display: grid;
-  grid-template-areas: "left center right";
+  grid-template-areas: "info nav selector";
   grid-template-rows: auto;
   grid-template-columns: 1fr auto 1fr;
   align-items: center;
   margin: 0 auto;
 }
 
+// Responsive layout
+// max-width is based on where component layout potentially breaks down
+@media screen and (max-width: 1000px) {
+  .hds-pagination {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .hds-pagination-nav {
+    justify-content: center;
+    order: -1;
+    width: 100%;
+  }
+
+  .hds-pagination-info,
+  .hds-pagination-size-selector {
+    margin:
+      var(--token-pagination-child-spacing-vertical)
+      var(--token-pagination-child-spacing-horizontal)
+      0
+      var(--token-pagination-child-spacing-horizontal);
+  }
+}
```

Comment:
I'm grouping all the responsive styles together into one media query for now until we decide which layout we want for narrower screens, then I will optimize the code more.